qdax/core/emitters/mees_emitter.py

```diff
@@ -0,0 +1,613 @@
+from dataclasses import dataclass
```

Comment:
Generally speaking, I am wandering if it wouldn't be cleaner to having an emitter for explore, another for exploit and have a meta emitter to create the explore/exploit. (not sure it's easy, just wandering)

What do you think about it?

Reply:
and obviously, we would take advantage of inheritance to have very small code repetition (or no code repetition at all)